src/stores/ProjectDuplicationStore.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios'
+import { defineStore } from 'pinia'
+
+export const useProjectDuplicationStore = defineStore({
```

Comment:
We don't need a store here since most of this information is only related to the single duplication request page.

The purpose of a store is to share items across pages. For example, 'taxa' is also shared on the specimens page so that we don't pull taxa information again. For pages which are only single use and their content should be not be cached, let's request it every time the page is loaded via `onMounted`.